packages/slice-machine/components/Changelog/index.tsx

```diff
@@ -30,42 +31,36 @@ export default function Changelog() {
     setSelectedVersion(latestVersion);
   }, [latestVersion]);
 
-  return !isChangelogLoading ? (
-    <Flex
-      sx={{
-        maxWidth: "1224px",
-      }}
-    >
-      <Navigation
-        changelog={changelog}
-        selectedVersion={selectedVersion}
-        selectVersion={(version) => setSelectedVersion(version)}
-      />
+  return (
+    <AppLayout>
+      <AppLayoutContent>
+        <BaseStyles sx={{ display: "flex", minWidth: 0 }}>
+          {!isChangelogLoading ? (
+            <>
+              <Navigation
+                changelog={changelog}
+                selectedVersion={selectedVersion}
+                selectVersion={(version) => setSelectedVersion(version)}
+              />
 
-      {changelog.sliceMachine.versions.length === 0 || !selectedVersion ? (
-        <Flex
-          sx={{
-            width: "754px",
-            minWidth: "754px",
-            height: "100%",
-            borderRight: "1px solid",
-            borderColor: "grey01",
-            flexDirection: "column",
-            padding: "24px 32px",
-            gap: "24px",
-          }}
-        >
-          <ReleaseWarning />
-        </Flex>
-      ) : (
-        <VersionDetails
-          changelog={changelog}
-          selectedVersion={selectedVersion}
-          packageManager={packageManager}
-        />
-      )}
-    </Flex>
-  ) : (
-    <LoadingPage />
+              {changelog.sliceMachine.versions.length === 0 ||
```

Comment:
For readability can we have brackets around the conditions?  
`(changelog.sliceMachine.versions.length === 0 || !selectedVersion) ? ...`

Reply:
yes :) thanks

Reply:
thanks :)